product_supplierinfo_group/views/product.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="product_template_tree_view" model="ir.ui.view">
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_only_form_view"/>
+        <field name="arch" type="xml">
+            <field name="seller_ids" position="replace">
```

Comment:
Hi, you should avoid using replace as it can break other module set invisible instead

Moreover there is two field seller_ids and variant_seller_ids we should hide both see here

https://github.com/OCA/OCB/blob/12.0/addons/purchase/views/product_views.xml#L14